.pipelines/e2e.yml

```diff
@@ -21,46 +23,40 @@ jobs:
   - template: ./templates/template-az-cli-login.yml
     parameters:
       azureDevOpsJSONSPN: $(aro-v4-e2e-devops-spn)
-  - script: |
-      az account set -s $AZURE_SUBSCRIPTION_ID
-      export SECRET_SA_ACCOUNT_NAME=$(SECRET_SA_ACCOUNT_NAME)
-      make secrets
-      . secrets/env
-      echo "##vso[task.setvariable variable=RP_MODE]$RP_MODE"
-    displayName: 🔑 Downloading certificates and secrets from storage account
-    name: setEnv
   - template: ./templates/template-push-images-to-acr.yml
     parameters:
       rpImageACR: $(RP_IMAGE_ACR)
       buildCommand: publish-image-aro
   - script: |
-      set -e
-      set -o pipefail
-
-      . secrets/env
-      export HIVEKUBECONFIGPATH="secrets/e2e-aks-kubeconfig"
-
-      az account set -s $AZURE_SUBSCRIPTION_ID
-
-      set -x
-
-      export PRIVATE_CLUSTER=true
-
+      export CI=true
       . ./hack/e2e/run-rp-and-e2e.sh
-      trap 'set +e; kill_rp; clean_e2e_db; kill_vpn' EXIT
```

Comment:
if we remove this trap, when does kill_vpn , kill_rp, clean_e2e_db get called ?

Reply:
I expect that those things will need to be running in order to collect the must-gather on failure so it would make the most sense to move them to a step that runs after everything regardless of success or failure by [using an `always()` condition](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/conditions?view=azure-devops&tabs=yaml).

Reply:
This step got lost during the rebase. It should be fixed now - running e2e again to see.